app/pages/Watching/index.js

```diff
@@ -54,31 +129,101 @@ class Watching extends Component {
     }
   };
 
+  onReset = async () => {
+    const {isConfirmingReset} = this.state;
+    const {network, reset} = this.props;
+
+    if (isConfirmingReset) {
+      await reset(network);
+      this.setState({isConfirmingReset: false});
+    } else {
+      this.setState({isConfirmingReset: true});
+    }
+  };
+
   onClose = () => this.setState({isAddingName: false, name: ''});
 
+  getText = () => {
+    const {
+      isConfirmingReset,
+      isImporting,
+    } = this.state;
+
+    if (isConfirmingReset) {
+      return 'This will permanently erased your watchlist on your device!';
```

Comment:
plural 'erase**d**'

suggest:
'Unless previously exported, this will permanently delete your watchlist.'